dev/AppNotifications/AppNotificationUtility.cpp

```diff
@@ -245,8 +245,10 @@ void Microsoft::Windows::AppNotifications::Helpers::RegisterAssets(std::wstring
     wil::unique_prop_variant propVariantDisplayName;
     // Do not throw in case of failure. We can use the filepath approach below as fallback to set a DisplayName.
     LOG_IF_FAILED(propertyStore->GetValue(PKEY_AppUserModel_RelaunchDisplayNameResource, &propVariantDisplayName));
-
```

Comment:
@danielayala94  Should the above call be a THROW_IF_FAILED?

Reply:
Yes. My logic was to prevent throwing if there was no element in the property store. But such assumption is wrong, because it returns S_OK anyway.

I will address this as part of #2228.